packages/expo-clipboard/src/Clipboard.types.ts

```diff
@@ -38,3 +38,32 @@ export interface ClipboardImage {
     height: number;
   };
 }
+
+export enum StringFormat {
+  PLAIN_TEXT = 'plainText',
+  HTML = 'html',
+}
+
+export interface GetStringOptions {
+  /**
+   * The target format of the clipboard string to be converted to, if possible.
+   * Defaults to plain text.
```

Comment:
We have `@default` annotation for that. Let's limit redundancy as much as possible.
```suggestion
```